test/stringify.js

```diff
@@ -535,4 +535,18 @@ test('stringify()', function (t) {
         );
         st.end();
     });
+
+    t.test('doNotEncodeKeys', function (st) {
+        st.equal(qs.stringify({ a: 'b', c: ['d', 'e=f'], f: [['g'], ['h']] }, { doNotEncodeKeys: true })
+        , 'a=b&c[0]=d&c[1]=e%3Df&f[0][0]=g&f[1][0]=h');
+        st.equal(qs.stringify({ a: 'b', c: ['d', 'e'], f: [['g'], ['h']] })
```

Comment:
if the entire function invocation can't fit on one line, then every argument must be on a line by itself, as well as the closing `)`.